scripts/area_plot.py

```diff
@@ -0,0 +1,613 @@
+import argparse
+import warnings
+# from warnings import Warning
+
+from matplotlib import rcParams
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+"""
+A set of python scripts to generate an area plot.
+"""
+# Sets up the matplotlib paramters so that we can save to be edited in 
```

Comment:
```suggestion
# Sets up the matplotlib parameters so that we can save to be edited in 
```

Also, please include an explanation of what `42` is in the comment

Reply:
I don't know the significance of 42 in this context. (Nor in Douglas Adams, but that's another problem entirely.)
I just know it's the magic number to get my matplotlib pdfs to behave nicely in adobe.